nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
As I've discovered, it also drops all earlier options if not given them again. 
-- 
Nikolay. 

Sent from a mobile device, please excuse my brevity.